### ISSUE TITLE
Emphasise that action payloads are also frozen

### ISSUE THREAD

**Issue by AnimaMundi** [brandonroberts/ngrx-store-freeze#11]
It is unclear when reading the readme that action payloads are frozen as well as the store itself.
I think this is something quite crucial and should be mentioned in the readme so as not to cause any confusion when using this module.

**Comment by changalberto** [comment]
Does anyone know how to circumvent the error thrown by mutated payload in this example:

![image](https://cloud.githubusercontent.com/assets/24946/23413855/0d514c3a-fda8-11e6-848e-a2665f33b78c.png)

```javascript
  @Effect()
  login$: Observable<Action> = this.actions$
    .ofType(AuthActions.ActionTypes.LOGIN)
    .map((action) => action.payload)
    .switchMap(credential => {
      if(!!!credential) {
        return empty();
      }

      return fromPromise(<Promise<any>>this.auth.login(credential))
        .map((response) => new AuthActions.LoginCompleteAction(response))
        .catch((err) => of(new AuthActions.LoginFailedAction(err)));
    });
```

**Comment by AnimaMundi** [comment]
@changalberto this seems like a question for stackoverflow, and it's hard to know for sure without seeing your authActions service but it looks like you are calling 'new' on a method of that authActions class. Which you don't need to do, the class method should return an action with normal invocation without the 'new' keyword. So if your authActions service is created in the way that I imagine, then removing 'new' should fix your problem.

**Comment by changalberto** [comment]
@AnimaMundi I'm simply following the conventions of this [example](https://github.com/ngrx/example-app/blob/0bd88242cadf243bdbec581417a833b5add6d14a/src/app/effects/book.ts). The actions in this example are classes.

**Comment by AnimaMundi** [comment]
@changalberto Ahh I followed a different method actually, I use functions as my action creators rather than classes. I naively assumed you were as well in my answer. I haven't seen this error before and it's hard to know what's going on without seeing more code, but I'd highly recommend taking this to StackOverflow. You'll get a lot more eyes on your issue there and likely someone who has seen it before will be able to help out.

**Comment by changalberto** [comment]
@AnimaMundi no worries. This is a unique situation. I don't think many people in the Ng2 community follow this architecture, but thanks anyway. I guess I'll go try my luck with StackOverflow.

**Comment by brandonroberts** [comment]
Fixed via f77d3885e914579059911c3eb4ea2f482ab96a9b . The items being frozen has been added to the README
